zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/tasks/BackgroundTask.java

```diff
@@ -11,4 +11,12 @@
 
 public interface BackgroundTask {
   CompletionStage<Integer> execute();
+
+  default String getCaption() {
+    final String simpleName = getClass().getSimpleName();
+    if (simpleName == null || simpleName.isEmpty()) {
+      return getClass().getName();
+    }
+    return simpleName;
+  }
```

Comment:
:wrench: imo we don't overload it with much benefit, I would just directly use `getClass().getSimpleName()`. I don't think simple name can ever be null though - but it could be empty if it's an unnamed class. I think this is only for anonymous classes though:

> Returns the simple name of the underlying class as given in the source code. An empty string is returned if the underlying class is `#isAnonymousClass()` anonymous or `#isUnnamedClass()` unnamed. A `#isSynthetic()` synthetic class, one not present in source code, can have a non-empty name including special characters, such as `$`.

So I think it's fine to just use `getClass().getSimpleName()` directly.

Reply:
Did you decide not to apply this?

Reply:
Sorry, wanted to apply, but forgot 🤦

Reply:
Basically I observed nulls in test for inner classes, but I agree, it does not worth keeping this check only for those rare test occasions.